### ISSUE TITLE
Filter 'Source name' based on 'Scene name'

### ISSUE THREAD

**Issue by jmcc94** [bitfocus/companion-module-obs-studio#90]
When using either an action or feedback that is based on a source within a scene ('Toggle visibility scene item' or 'Change colors when source enabled in scene', the scene dropdown shows all possible sources within OBS.

What would be useful if it was filtered based on the scene chosen in the other dropdown (or if no scene is chosen, then all sources within the current scene'

**Comment by bryce-seifert** [comment]
I was kind of thinking this too. I'll take a look at it.

**Comment by bryce-seifert** [comment]
This feature is currently waiting on development of this [core feature](https://github.com/bitfocus/companion/issues/1635)

**Comment by bryce-seifert** [comment]
Yes the issue here is with dynamically updating the dropdown. We have all the data on what sources are in a scene, but not a way to update it currently
